astrolabe/utils.py

```diff
@@ -285,7 +286,7 @@ def get_logs(admin_client, project, cluster_name):
     params = dict(
         resourceName=rname,
         resourceType=rtype,
-        redacted=True,
+        redacted=False,
```

Comment:
this was changed to workaround the cloud bug where this option led to unavailability to use server logs at all

Reply:
Same with this, can you please add a note in the source code stating that redaction on 4.4 servers in Atlas produces garbled log files with a reference to the help ticket where this is determined and the cloudp ticket that would resolve the problem.

Reply:
done